e2e/porch_test.go

```diff
@@ -108,6 +113,25 @@ func runTestCase(t *testing.T, git string, tc porch.TestCaseConfig) {
 	}
 }
 
+func reorderYamlStdout(t *testing.T, buf *bytes.Buffer) {
+	var data interface{}
+	if err := yaml.Unmarshal(buf.Bytes(), &data); err != nil {
+		// not yaml.
+		return
+	}
+
+	var stable bytes.Buffer
+	encoder := yaml.NewEncoder(&stable)
```

Comment:
I am little nervous using another yaml serializer. Should we stick to "sigs.k8s.io/kustomize/kyaml/yaml" ?

Reply:
actually, sigs.k8s.io/kustomize/kyaml/yaml may do since it uses (a clone of an earlier version) of the same module. The deterministic ordering was introduced in this yaml encoder in v2 so with some luck kyaml inherited it.

Reply:
kyaml fork of go-yaml works.